build/common/installer/scripts/tomlparser.rb

```diff
@@ -150,6 +151,21 @@ def populateSettingValuesFromConfigMap(parsedConfig)
       ConfigParseErrorLogger.logError("Exception while reading config map settings for container log schema version - #{errorStr}, using defaults, please check config map for errors")
     end
 
+    # Get multiline log enabling setting
+    begin
+      if !parsedConfig[:log_collection_settings][:enable_multiline_logs].nil? && !parsedConfig[:log_collection_settings][:enable_multiline_logs][:enabled].nil?
```

Comment:
I didnt see the configmap updates for this, is this intentional?

Reply:
added